frontend/js/src/recent/RecentListens.tsx

```diff
@@ -248,6 +255,18 @@ export default class RecentListens extends React.Component<
                       />
                     );
                   }
+                  additionalMenuItems.push(
```

Comment:
This version is half-implemented. I initially didn't realise that there was a separate implementation of the listen card/menu in RecentListens. My recommendation is that we remove this.

Reply:
Added to LB-1179